core/src/test/java/io/narayana/nta/test/CentralisedAS8IntegrationTest.java

```diff
@@ -76,6 +76,7 @@ public static WebArchive createDeployment() {
                 .addAsWebInfResource(new FileAsset(new File("src/test/resources/persistence.xml")),
                         "classes/META-INF/persistence.xml")
                 .addAsManifestResource(new FileAsset(new File("src/test/resources/nta-test-ds.xml")), "nta-test-ds.xml")
+	        .addAsManifestResource(new FileAsset(new File("src/test/resources/filter.properties")), "filter.properties") 
```

Comment:
I think the filter.properties will be added into a test.war and deployed into the container during the aruquillian running the tests. I do not understand how to use the class loader here.